meta-facebook/yv4-sd/src/platform/plat_pldm_monitor.c

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Majority of the code in pldm_monitor.h and pldm_monitor.c can be moved into common layer as common pldm code so other platforms can use it too. Platform specific code in this case should just have bunch of tables with platform specific info. Can you please revisit this changeset from that perspective and make necessary changes?